plugins/module_utils/network/sonic/config/copp/copp.py

```diff
@@ -425,3 +446,12 @@ def get_copp_groups_key(self, copp_groups_key):
     def sort_lists_in_config(self, config):
         if 'copp_groups' in config and config['copp_groups'] is not None:
             config['copp_groups'].sort(key=self.get_copp_groups_key)
+
+    def get_error_msg(self, want, state):
```

Comment:
When error, this function will not return. So it may be better to name it like "validate_want_for_replaced_overridden".